onnxruntime/core/providers/openvino/ov_versions/capability_2021_1.cc

```diff
@@ -208,6 +210,35 @@ bool IsOpSupported(std::string name, std::string device) {
         supported_ops.insert(supported_ops_cpu.begin(), supported_ops_cpu.end());
      }
     }
+  } else if (device.find("MULTI") == 0) {
+    std::vector<std::string> devices;
+    std::stringstream s_stream(device);
+    while(s_stream.good()) {
+      std::string substr;
+      getline(s_stream, substr, ',');
+      devices.push_back(substr);
+    }
+    if (common_supported_ops.count(name)) {
```

Comment:
the count() function of std::set, which is the type of common_supported_ops supported_ops_*pu, returns a value of type size_type which is an unsigned integer. It is not a clean practice to use it as an expression of if,else statements which need to be boolean. Use expressions like "common_supported_ops.count(name) == 0" instead

Reply:
Done @smkarlap